scripts/publish-local.sh

```diff
@@ -28,8 +28,8 @@ docker run -it --rm \
       -e GDALCFLAGS="-I/macintosh/gdal/3.1.2/include" \
       -e CXXFLAGS="-I/usr/osxcross/SDK/MacOSX10.10.sdk/usr/include/c++/v1"  \
       -e BOOST_ROOT="/usr/local/include/boost_1_69_0" \
-      -e LDFLAGS="-mmacosx-version-min=10.9 -L/macintosh/gdal/3.1.2/lib -lgdal -lstdc++ -lpthread" \
-      daunnc/gdal-build-environment:6 make -j4 -C src libgdalwarp_bindings.dylib || exit -1
+      -e LDFLAGS="-mmacosx-version-min=10.9 -L/macintosh/gdal/3.1.2/lib -lgdal -lstdc++ -lpthread -Wl,-rpath,/usr/local/lib" \
+      jamesmcclain/gdal-build-environment:6 make -j4 -C src libgdalwarp_bindings.dylib || exit -1
```

Comment:
The most quick solution would be to use your dockerhub account (i.e. `cloudniner/gdal-build-environment:7`); otherwise you need quay azavea credentials, but the release process still should be done from your local machine (or some EC2 machine);

Don't forget the CircleCI config as well: https://github.com/geotrellis/gdal-warp-bindings/blob/master/.circleci/config.yml#L6

Reply:
🎉 thanks. Updated.